### PR TITLE
Fixing clipboard for MacOS

### PR DESCRIPTION
**Pull request by Heychsea**
Small fix to enable Pasteboard load on MacOS.

### CODE REVIEW

File: src/master/mac.rs [hunk 1]
```diff
@@ -4,6 +4,9 @@ use crate::{ClipboardHandler, CallbackResult, Master};
 use objc::runtime::{Object, Class};
 use objc_id::{Id};
 
+#[link(name = "AppKit", kind = "framework")]
```
Comment (DoumanAsh) [inline src/master/mac.rs:7]: interesting, `objc` doesn't link it?

Comment (Heychsea) [reply src/master/mac.rs:7]: I'm not really an expert on this, but it wasn't linked until I used this on my mac (Big Sur 11.2). It is also used in many clipboard crates.

Comment (DoumanAsh) [reply src/master/mac.rs:7]: Understood

**Comment by DoumanAsh** [comment]
I've published 3.1.1 with this fix

**Pull request merged**